exercises/anagram/anagram_test.rb

```diff
@@ -3,70 +3,120 @@
 require 'minitest/autorun'
 require_relative 'anagram'
 
+# Test data version:
+# 3dfde31
 class AnagramTest < Minitest::Test
   def test_no_matches
-    detector = Anagram.new('diaper')
-    assert_equal [], detector.match(%w(hello world zombies pants))
+     # skip
+     assert_equal [], Anagram.new("diaper").match(["hello", "world", "zombies", "pants"])
   end
 
-  def test_detect_simple_anagram
-    skip
-    detector = Anagram.new('ant')
-    anagrams = detector.match(%w(tan stand at))
-    assert_equal ['tan'], anagrams
+  def test_detects_simple_anagram
+     skip
+     assert_equal ["tan"], Anagram.new("ant").match(["tan", "stand", "at"])
```

Comment:
Why have you changed from the old 3 line "detector, anagrams, assertion" format?